src/components/XAxis/XAxis.tsx

```diff
@@ -13,20 +14,28 @@ type Props = {
 	ticks?: { from?: From; to?: To; jumps?: Jumps };
 	title?: ReactNode;
 	description?: ReactNode;
+	horizontal?: boolean;
 };
 
-export const XAxis = ({ title, description }: Props) => {
+export const XAxis = ({ title, description, horizontal }: Props) => {
```

Comment:
No horizontal prop on this please.
Should be sufficient for someone who wants horizontal bars to:

1. Swap the x/y of their dataset around i.e {x: "January" becomes {y: "January" if you want horizontal.
2. use `<Bars horizontal={true}/>`


This removes the need for changes to XAxis/YAxis in ways that only make sense for <Bars/> but not <Lines/>

Reply:
done